### PR TITLE
fix: update dashboard

### PR DESCRIPTION
**Pull request by Krish120003**

### CODE REVIEW

**Comment by fvcci** [comment]
I don't see any changes from commit [beeaf3a](https://github.com/deltahacks/portal/pull/178/commits/beeaf3a2f5e16e7d21f8496581973365001a4d49) in the main branch

**Comment by fvcci** [comment]
image
I got a warning in console about a list not having a key. I think it has to do with grade.tsx:41?

**Comment by Krish120003** [comment]
@fvcci The key prop thing we can add to bug fix list. I'm not sure what happened to the commit you mentioned, but since its related to application submissions which are now closes, is that a priority to fix in this PR or can we delay that?

**Comment by fvcci** [comment]
> @fvcci The key prop thing we can add to bug fix list. I'm not sure what happened to the commit you mentioned, but since its related to application submissions which are now closes, is that a priority to fix in this PR or can we delay that?

what is a quote reply 😩. Since applications are closed I suppose it's not that necessary? I'll add them as issues on linear. One outstanding issue though would be disabling the discord button from dashboard.

**Comment by Krish120003** [comment]
Does this PR not do that?

File: src/pages/dashboard.tsx [hunk 1]
```diff
@@ -289,35 +310,30 @@ const RSVPed: React.FC = () => {
...
-          <button className="btn btn-primary w-48 border-none bg-zinc-700 text-base font-medium capitalize hover:bg-zinc-800">
+          </Link>
+        </Button>
+        <Button className="btn w-48 border-none hover: hover:bg-zinc-700 text-base font-medium capitalize">
+          <Link
+            className="w-full md:w-48"
+            href="https://discord.gg/22ddpvfwXn"
+            target="_blank"
+          >
             Discord
```
[the remark this answers is not in the document]
Comment (fvcci) [reply src/pages/dashboard.tsx:329]: The button to the discord doesn't need to be fixed. Looks like the link to the server was already changed to lead to the server we always use.

**Review by fvcci — APPROVED** [review]
The change looks good to me. Any outstanding issues like the invalid date bug or the exception where a list doesn't have a key have been filed on linear.

**Pull request merged**